platform/model/main.graphql

```diff
@@ -81,6 +83,11 @@ type VolunteerAction {
   distributionCentre: DistributionCentre
 }
 
+interface Location {
```

Comment:
My recommendation will be to change it to Address and also add 

- geo data lat long
- city
- postcode (optional)

Reply:
Let's change it to `Address` and add `city` and `postcode` for now, `lat` and `long` can be added later once we able to retrieve them from map component.